packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -351,6 +350,13 @@ class AndroidWebViewController extends PlatformWebViewController {
   Future<void> setUserAgent(String? userAgent) =>
       _webView.settings.setUserAgentString(userAgent);
 
+  @override
+  Future<void> setOnContentOffsetChanged(
+      void Function(int left, int top, int oldLeft, int oldTop)?
+          onOffsetChange) async {
+    _webView.onScrollChanged = onOffsetChange;
```

Comment:
Since `_webView.onScrollChanged` will be a `final` field, you can follow the pattern of `setOnShowFileSelector`. You can add a  field called `_OnContentOffsetChangedCallback` to the `AndroidWebViewController` and this field can be used when the `WebView` is instantiated.